### PR TITLE
Refactor FXIOS-8313 [Multi-window] DRY out hardcoded test window UUID

### PR DESCRIPTION
**Pull request by mattreaganmozilla**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8313)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18426)

## :bulb: Description

General clean-up. Part of larger refactor efforts for multi-window. 

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: BrowserKit/Tests/ReduxTests/Utilities/FakeReduxAction.swift [hunk 1]
```diff
@@ -19,7 +19,6 @@ enum FakeReduxAction: Action {
     case setPrivateModeTo(Bool)
 
     var windowUUID: UUID {
-        // TODO: Update to use static consts on WindowUUID (.XCTestDefaultUUID)
         return UUID(uuidString: "D9D9D9D9-D9D9-D9D9-D9D9-CD68A019860B")!
```
Comment (mattreaganmozilla) [inline BrowserKit/Tests/ReduxTests/Utilities/FakeReduxAction.swift:22]: Note: these Redux tests are isolated within BrowserKit so for now I'm just leaving this UUID hardcoded.

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Theme/ThemableTests.swift [hunk 2]
```diff
@@ -90,7 +90,6 @@ class TestsThemeable: UIViewController, Themeable {
 
     func applyTheme() {}
     var currentWindowUUID: UUID? {
-        // TODO: Use .XCTestDefaultUUID
         return UUID(uuidString: "D9D9D9D9-D9D9-D9D9-D9D9-CD68A019860B")!
```
Comment (cyndichin) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Theme/ThemableTests.swift:93]: return `.XCTestDefaultUUID`?

File: firefox-ios/Client/Frontend/Browser/Event Queue/AppEvent.swift [hunk 3]
```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Foundation
+import Shared
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Browser/Event Queue/AppEvent.swift:6]: needed?

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift [hunk 4]
```diff
@@ -4,11 +4,13 @@
 
 import XCTest
 import WebKit
+import Storage
```
Comment (cyndichin) [inline firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift:7]: needed?

**Review by cyndichin — APPROVED** [review]
some small nits, otherwise approved! thank you

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Theme/ThemableTests.swift (hunk 2 above)
Comment (mattreaganmozilla) [reply firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Theme/ThemableTests.swift:93]: Fixed, ty

File: firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift (hunk 4 above)
Comment (mattreaganmozilla) [reply firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTests.swift]: Fixed, ty

**Pull request merged**